packages/shared/src/lib/feedItems.shared.ts

```
@@ -0,0 +1,111 @@
+  }
+
+  public static getStarFeedItemActionInfo(feedItem: FeedItem): FeedItemAction {
+    const isAlreadyStarred = SharedFeedItemHelpers.isStarred(feedItem);
+    return {
+      type: FeedItemActionType.Star,
+      text: isAlreadyStarred ? 'Unstar' : 'Star',
+      icon: IconName.Star,
+      shortcutId: KeyboardShortcutId.ToggleStarred,
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)